firefox-ios/Client/Frontend/TabContentsScripts/LoginsHelper.swift

```diff
@@ -340,6 +341,15 @@ class LoginsHelper: TabContentScript, FeatureFlaggable {
         currentRequestId = requestId
     }
 
+    private func clearStoredPasswordPassGen() {
```

Comment:
```suggestion
    private func clearStoredPasswordPassGen() {
```

are you trying to say to clear it after generation if so a better terminology would be `clearStoredPasswordAfterGeneration`